.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
Let's change the name of this file to `past-events.yml`, and put a nice big comment at the top explaining what it does.  Then add a schedule trigger (once a week?), and we are good to go!

Reply:
What day and time should I set this for? I am currently testing it to run on mondays at 6:20pm just to test the on schedule action. Once that works, I can set it to a schedule we agree upon.

Reply:
It doesn't much matter.  How about the wee hours of Sunday morning? 
```
  schedule:
    # Run at 2:05am early Sunday morning Eastern time (6/7:05 UTC)
    # https://crontab.guru/#5_7_*_*_0
    - cron: "5 7 * * 0"
```

Reply:
Not sure what I am doing wrong with the syntax. I am not able to get the job scheduled. Earlier I was not using UTC time, but then realized I need to use UTC. Since then tried scheduling at 1UTC, then 1:16UTC and none of them ran. 

I read (https://upptime.js.org/blog/2021/01/22/github-actions-schedule-not-working/) that there are usually delays and scheduled time doesn't always start exactly on time. So will let this be over night and see what happens.

Reply:
Finally ran.. I have now set it up for the suggested 2:05AM sunday morning EST time.